src/org/apache/xerces/xni/XMLAttributes.java

```diff
@@ -139,7 +140,7 @@ public interface XMLAttributes {
     public void getName(int attrIndex, QName attrName);
 
     /**
-     * Returns the prefix of the attribute at the specified index.
+     * @return the prefix of the attribute at the specified index.
```

Comment:
no period in an @return that does not have a second phrase/sentence.

Reply:
fixed in next commit